cg/cli/deliver/base.py

```diff
@@ -0,0 +1,77 @@
+"""CLI for delivering files with CG"""
+
+import logging
+
+from typing import List
+from pathlib import Path
+
+import click
+
+from cg.meta.deliver import DeliverAPI
+from cg.store import Store
+from cg.store.models import Family
+from cg.apps.hk import HousekeeperAPI
+from cg.constants.delivery import (
+    PIPELINE_ANALYSIS_OPTIONS,
+    PROJECT_BASE_PATH,
+    PIPELINE_ANALYSIS_TAG_MAP,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+@click.pass_context
+def deliver(context):
+    """Deliver files with CG."""
+    LOG.info("Running CG deliver")
+    context.obj["store"] = Store(context.obj["database"])
+    context.obj["hk_api"] = HousekeeperAPI(context.obj)
```

Comment:
Could you call these objects "status_db" and "housekeeper_api"? It would be nice to have them named consistently thoughout the codebase!

Reply:
(also in tests)

Reply:
Ok

Reply:
We should use a enum-class for this